data/35e/wizards_of_the_coast/supplement/players_handbook_2/ph2_abilities.lst

```diff
@@ -8,7 +8,7 @@ Armored Mage			KEY:Beguiler ~ Armored Mage				CATEGORY:Special Ability	TYPE:Extr
-Advanced Learning			KEY:Beguiler ~ Advanced Learning			CATEGORY:Special Ability	TYPE:Extraordinary.Class Feature.Beguiler Class Feature.SpecialQuality	DEFINE:AdvancedLearningCount|0	DESC:At 3rd level, you can add a new spell to your list, representing the result of personal study and experimentation. The spell must be a sorcerer/ wizard spell of the enchantment or illusion school and of a level no higher than that of the highest-level spell you already know. Once a new spell is selected, it is forever added to your spell list and can be cast just like any other spell on your list. You gain another new spell at 7th, 11th, 15th, and 19th level.																																																																																																																																																																																	BONUS:ABILITYPOOL|Advanced Learning|AdvancedLearningCount	SOURCEPAGE:p.7		BONUS:VAR|AdvancedLearningCount|(BeguilerLVL+1)/4	DEFINE:BeguilerHighestLevelAllowed|0		BONUS:VAR|BeguilerHighestLevelAllowed|floor(BeguilerLVL/2)
+Advanced Learning			KEY:Beguiler ~ Advanced Learning			CATEGORY:Special Ability	TYPE:Extraordinary.Class Feature.Beguiler Class Feature.SpecialQuality	DEFINE:AdvancedLearningCount|0	DESC:At 3rd level, you can add a new spell to your list, representing the result of personal study and experimentation. The spell must be a sorcerer/ wizard spell of the enchantment or illusion school and of a level no higher than that of the highest-level spell you already know. Once a new spell is selected, it is forever added to your spell list and can be cast just like any other spell on your list. You gain another new spell at 7th, 11th, 15th, and 19th level.																																																																																																																																																																																	BONUS:ABILITYPOOL|Advanced Learning|AdvancedLearningCount	SOURCEPAGE:p.7		BONUS:VAR|AdvancedLearningCount|(BeguilerLVL+1)/4	DEFINE:BeguilerHighestLevelAllowed|0		BONUS:VAR|BeguilerHighestLevelAllowed|floor(CASTERLEVEL/2)
```

Comment:
Have you confirmed this actually works?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/425183/149682652-bc8520e6-c660-4fda-a68c-eba164b7706f.png)

Reply:
She is Beguiller with a dip into Mindbender